aspnet/fundamentals/app-state/sample/src/AppState/Startup.cs

```diff
@@ -48,7 +43,7 @@ public void ConfigureServices(IServiceCollection services)
                 subApp.Run(async context =>
                 {
                     await context.Response.WriteAsync("<html><body>");
-                    await context.Response.WriteAsync("Requested at: " + DateTime.Now.ToLongTimeString() + "<br>");
```

Comment:
I assume you're changing these to allow compatibility with .NET Core, correct? If so, that's fine.